components/base/DHero.vue

```diff
@@ -1,5 +1,11 @@
 <template>
   <section
+    v-if="
+      $route.params.main === 'services' ||
+      $route.params.main === 'our-work' ||
```

Comment:
Should we switch the if logic so that it looks for home in the positive case, then falls through to the plain banner?  This is probably safer maintenance-wise if we add more sections to the web site

Reply:
also, it looks like the only difference between the if and the else is the `header-image` class, can that just be added to the `:class` binding somehow instead?